examples/01_plotting/plot_demo_glass_brain.py

```diff
@@ -20,10 +20,19 @@
 # demo glass brain plotting
 from nilearn import plotting
 
+# Whole brain saggital cuts
 plotting.plot_glass_brain(localizer_tmap_filename, threshold=3)
 
 plotting.plot_glass_brain(
     localizer_tmap_filename, title='plot_glass_brain',
     black_bg=True, display_mode='xz', threshold=3)
 
+# Hemispheric saggital cuts
+plotting.plot_glass_brain(
+    localizer_tmap_filename, display_mode='lyrz', threshold=3)
+
+plotting.plot_glass_brain(
```

Comment:
We want to keep this example as simple as possible. I suggest that we remove that last line, which belongs more to the advanced example.